src/ctapipe/image/muon/features.py

```diff
@@ -334,16 +334,14 @@ def radial_light_distribution(
     delta_r = pixel_r - mean
     radial_std_dev = np.sqrt(np.average(delta_r**2, weights=image))
     skewness = (
-        np.average(delta_r**3, weights=image)
-        / radial_std_dev**3
-        * u.dimensionless_unscaled
-    )
+        np.average(delta_r**3, weights=image) / radial_std_dev**3
+    ) * u.dimensionless_unscaled
```

Comment:
Why attach `u.dimensionless_unscaled` to remove it later?

Wouldn't the better fix be just to make sure here the computed quantity is already unit-less?

Reply:
See e.g. how `all_to_unit` is used in `hillas_parameters`